rotkehlchen/chain/ethereum/modules/defisaver/constants.py

```diff
@@ -7,3 +7,32 @@
 
 SUBSCRIBE: Final = b'w\x91\xee\xa7\xfdZ\x87y@\xfd\x06T\xc6\xb1\xe0Y\x11)<\xed\xab\xdf\xe1\xf9\x9b|,\xb00\x8d\xba\xf7'  # noqa: E501
 DEACTIVATE_SUB: Final = b'\x1f\x854\xa3*\xe8\x1aG\xc9\xe9\xfe%m\x92O:\xd6h\xe9\x19\x16\xa4\x8f\xd4\x1a\xc2\x7f\x84\xc6m\x94,'  # noqa: E501
+
+DFS_GENERIC_LOGGER_LEGACY: Final = string_to_evm_address('0x5c55B921f590a89C1Ebe84dF170E655a82b62126')  # noqa: E501
+
+# FLAction contract addresses taken from https://github.com/defisaver/defisaver-v3-contracts/blob/main/addresses/mainnet.json
```

Comment:
is there any on chain registry for this?

Reply:
I can ask the question to them, but in past they suggested me to look at this repo for all their contracts

Reply:
there is a registry address ( 0x287778f121f134c66212fb16c9b53ec991d32f5b ) and this method: 

![immagine](https://github.com/user-attachments/assets/951b0ce9-3098-4ac9-85e2-064c5b3cecad)

Is this what you were looking for?

Reply:
If they don't change often is fine to have them hardcoded. We use on chain registries if there are new ones often that need to be tracked fast.